docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -121,7 +121,45 @@ def continent_job():
 
 It's common that, when you have a partitioned job, you want to run it on a schedule. For example, if your job has a partition for each date, you likely want to run that job every day, on the partition for that day.
 
-The <PyObject object="build_schedule_from_partitioned_job"/> function allows you to construct a schedule from a partitioned job. The [Schedules concept page](/concepts/partitions-schedules-sensors/schedules#a-schedule-from-a-partitioned-job) describes how to use it.
+The <PyObject object="build_schedule_from_partitioned_job"/> function allows you to construct a schedule from a date partitioned job. If you wanted to create a schedule for `do_stuff_partitioned` job defined above, you could write:
```

Comment:
maybe also add something that explicitly states that the cron schedule is inferred from the partition scheme when using this helper function?